src/components/CreateMsa.svelte

```diff
@@ -62,14 +77,15 @@
   const clearTxnStatuses = () => (txnStatuses = new Array<string>());
 </script>
 
-<div id="create-msa" class="action-card basis-1/2">
+<div id="create-msa" class="flex flex-col gap-3.5 text-sm">
+  <div class="label">Create MSA Id</div>
   <p>
     An MSA (Message Source Account) is required to become a provider. This action will create an MSA Id that is
     controlled by the selected Transaction Signing Address above. It is available only on Frequency Testnet.
   </p>
-  <form class="w-350 flex justify-between">
+  <form class="w-350 flex items-end justify-between">
```

Comment:
Checking that w-350 is actually doing something. Curious if this was meant to be w-[350px]. Are we reusing this width? Should we define it in tailwind.config?

Reply:
This was in code prior to my PR.  I replaced all occurrences of `w-350` with `w-[350px]`